platforms/native/engine/jsb-spine-skeleton.js

```diff
@@ -783,6 +719,70 @@ const cacheManager = require('./jsb-cache-manager');
         const nativeSkeleton = this._nativeSkeleton;
         if (!nativeSkeleton) return;
 
+        if (!this.isAnimationCached() && (this.debugBones || this.debugSlots || this.debugMesh) && this._debugRenderer) {
```

Comment:
Why move to here

Reply:
> Why move here
updateAnimation call by skeletonSystem it's ahead of debugBones or debugMesh setting.